google-cloud-bigquerystorage/src/main/java/com/google/cloud/bigquery/storage/v1/JsonStreamWriter.java

```diff
@@ -336,6 +337,22 @@ public Builder setLocation(String location) {
       return this;
     }
 
+    /**
+     * Sets the compression to use for the calls. The compressor must be of type gzip.
+     *
+     * @param compressorName
+     * @return Builder
+     */
+    public Builder setCompressorName(String compressorName) {
+      Preconditions.checkNotNull(compressorName);
+      Preconditions.checkArgument(
+          compressorName.equals("gzip"),
+          "Compression of type \"%s\" isn't supported, only \"gzip\" compression is supported.",
```

Comment:
Only check it at the bottom layer (ConnectionWorker) is enough?

Reply:
Removed from JsonStreamWriter and SchemaAwareStreamWriter, and left it in StreamWriter. Verified that it's what we usually do.